start_cerberus.py

```diff
@@ -78,6 +78,9 @@ def main(cfg):
             logging.info("Detailed inspection of failed components has been enabled")
             inspect.delete_inspect_directory()
 
+        # get list of all master nodes
+        master_nodes = kubecli.list_master_nodes()
```

Comment:
Maybe call the function ( https://github.com/openshift-scale/cerberus/blob/master/cerberus/kubernetes/client.py#L19 ) with node-role.kubernetes.io=master as the label selector to get the list of the master nodes?